src/encoded/static/components/item-pages/MicroscopeConfigurationView.js

```diff
@@ -0,0 +1,542 @@
+                            'title': "Updated Status for " + confTitle,
+                            'message': (
+                                <p className="mb-02">
+                                    Changed Display status to <b>{statusToSet}</b>.
+                                    It may take some time for this edit to take effect.
+                                </p>
+                            ),
+                            'style': 'info'
+                        });
+                    },
+                    'PATCH',
+                    (resp) => {
+                        // Error callback
+                        this.setState({ 'releaseLoading': false });
+                        Alerts.queue({
+                            'title': "Failed to release display.",
+                            'message': "Sorry, can you try to share again?",
+                            'style': 'danger'
+                        });
+                    },
+                    JSON.stringify({
+                        'status': statusToSet
+                    })
+                );
+            }
+        );
+    }
+
+    statusChangeButton(){
+        const { session, context } = this.props;
+        const { saveLoading, cloneLoading, releaseLoading } = this.state;
+        const editPermission = this.havePermissionToEdit();
+
+        if (!session || !editPermission) return null; // TODO: Remove and implement for anon users. Eventually.
+
+        const btnProps  = {
+            'onSelect'      : this.handleStatusChange,
+            //'onClick'       : context.status === 'released' ? null : this.handleStatusChangeToRelease,
+            'variant'       : context.status === 'released' ? 'outline-dark' : 'info',
+            'disabled'      : releaseLoading,
+            'key'           : 'statuschangebtn',
+            'data-tip'      : "Change the visibility/permissions of this Microscope Configuration",
+            'title'         : (
+                <React.Fragment>
+                    <i className={"icon icon-fw icon-" + (releaseLoading ? 'circle-notch fas icon-spin' : 'id-badge far')}/>&nbsp; Manage
+                </React.Fragment>
+            ),
+            'pullRight'     : true
+        };
+
+        return (
+            <DropdownButton {...btnProps}>
+                <StatusMenuItem eventKey="released" context={context}>Visible by Everyone</StatusMenuItem>
+                <StatusMenuItem eventKey="released to project" context={context}>Visible by Network</StatusMenuItem>
+                <StatusMenuItem eventKey="released to lab" context={context}>Visible by Lab</StatusMenuItem>
+                <StatusMenuItem eventKey="draft" context={context}>Private</StatusMenuItem>
+                <Dropdown.Divider />
+                <StatusMenuItem eventKey="deleted" context={context}>Delete</StatusMenuItem>
+            </DropdownButton>
+        );
+    }
+
+    saveButton(){
+        const { session, context } = this.props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['context' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4271272)